### PR TITLE
Tweaked grammar in shaders_basic_pbr.c

### PR DESCRIPTION
**Pull request by Mute124**
All that was done is fixing some of the grammar within the comments. Other than that, nothing else is changed

-Mute

### CODE REVIEW

File: examples/shaders/shaders_basic_pbr.c [hunk 1]
```diff
@@ -94,9 +94,10 @@ int main()
     camera.fovy = 45.0f;                                // Camera field-of-view Y
     camera.projection = CAMERA_PERSPECTIVE;             // Camera projection type
 
-    // Load PBR shader and setup all required locations
+    // Load the PBR shader and setup all required locations
```
Comment (raysan5) [inline examples/shaders/shaders_basic_pbr.c:97]: I'm afraid I'm not using `the` in any of the shaders examples, for consitency, I prefer to keep it without it.

File: examples/shaders/shaders_basic_pbr.c [hunk 2]
```diff
@@ -105,11 +106,11 @@ int main()
     shader.locs[SHADER_LOC_MAP_NORMAL] = GetShaderLocation(shader, "normalMap");
     // WARNING: Similar to the MRA map, the emissive map packs different information 
     // into a single texture: it stores height and emission data
-    // It is binded to SHADER_LOC_MAP_EMISSION location an properly processed on shader
+    // It is binded to the SHADER_LOC_MAP_EMISSION location and properly processed on the shader
```
Comment (raysan5) [inline examples/shaders/shaders_basic_pbr.c:109]: I'm afraid I'm not using `the` in any of the shaders examples, for consitency, I prefer to keep it without it.

File: examples/shaders/shaders_basic_pbr.c [hunk 3]
```diff
@@ -105,11 +106,11 @@ int main()
...
     shader.locs[SHADER_LOC_MAP_EMISSION] = GetShaderLocation(shader, "emissiveMap");
     shader.locs[SHADER_LOC_COLOR_DIFFUSE] = GetShaderLocation(shader, "albedoColor");
 
-    // Setup additional required shader locations, including lights data
+    // Setup additional required shader locations, including the lights data
```
Comment (raysan5) [inline examples/shaders/shaders_basic_pbr.c:113]: I'm afraid I'm not using `the` in any of the shaders examples, for consitency, I prefer to keep it without it.

File: examples/shaders/shaders_basic_pbr.c [hunk 4]
```diff
@@ -122,7 +123,7 @@ int main()
     SetShaderValue(shader, GetShaderLocation(shader, "ambientColor"), &ambientColorNormalized, SHADER_UNIFORM_VEC3);
     SetShaderValue(shader, GetShaderLocation(shader, "ambient"), &ambientIntensity, SHADER_UNIFORM_FLOAT);
 
-    // Get location for shader parameters that can be modified in real time
+    // Get the location for shader parameters that can be modified in real time
```
Comment (raysan5) [inline examples/shaders/shaders_basic_pbr.c:126]: I'm afraid I'm not using `the` in any of the shaders examples, for consitency, I prefer to keep it without it.

File: examples/shaders/shaders_basic_pbr.c (hunk 2 above)
Comment (orcmid) [reply examples/shaders/shaders_basic_pbr.c:109]: Not using "the" is fine.  However "binded" is not a word :).  I believe the correct term is "bound."

Comment (Mute124) [reply examples/shaders/shaders_basic_pbr.c:109]: > Not using "the" is fine. However "binded" is not a word :). I believe the correct term is "bound."

Shoot you are right! my bad

**Comment by raysan5** [comment]
I'm closing this PR, feel free to open another one with the grammar fixes if desired.

**Pull request closed without merging**